packages/react/src/lib/accessibility/FocusZone/AutoFocusZone.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react'
+import * as PropTypes from 'prop-types'
+import * as _ from 'lodash'
+
+import { getNextElement, focusAsync } from './focusUtilities'
+
+import { AutoFocusZoneProps } from './AutoFocusZone.types'
+import getUnhandledProps from '../../getUnhandledProps'
+import getElementType from '../../getElementType'
+import * as customPropTypes from '../../customPropTypes'
+import { callable } from '../../index'
```

Comment:
```suggestion
import callable from '../../callable'
```

To simplify imports resolution